smart_card_connector_app/src/window-main.js

```diff
@@ -58,7 +58,34 @@ GSC.ConnectorApp.Window.AppsDisplaying.initialize();
 GSC.ConnectorApp.Window.DevicesDisplaying.initialize();
 GSC.ConnectorApp.Window.LogsExporting.initialize();
 
-GSC.PopupWindow.Client.setExplicitWindowTitle();
 GSC.PopupWindow.Client.showWindow();
 
+/** @const */
+var DATA_I18N_STR = 'data-i18n';
+
+/** @const */
+var DATA_I18N_ARIA_LABEL_STR = 'data-i18n-aria-label';
+
+function setElementI18nText(element) {
+  var i18nId = element.getAttribute(DATA_I18N_STR);
+  var translatedText = chrome.i18n.getMessage(i18nId);
+  element.innerText = translatedText;
+}
+
+function setElementI18nAriaLabel(element) {
+  var i18nId = element.getAttribute(DATA_I18N_ARIA_LABEL_STR);
+  var translatedText = chrome.i18n.getMessage(i18nId);
+  element.setAttribute('aria-label', translatedText);
+}
+
+for (let element of /** @type {?} */ (goog.dom.getElementsByTagNameAndClass(
```

Comment:
Have you found anywhere in the docs that goog.dom.getElementsByTagNameAndClass may accept such selectors. I think it's just a coincidence that it works that way (actually, it contains several branches, and it was just a luck that the picked branch uses Element.querySelectorAll instead of any other way).

So, instead of relying on this behavior (which may change at any time), some cleaner way should be used - e.g. using Document.querySelectorAll.